index.html

```diff
@@ -820,8 +820,13 @@ <h3>
           <li>[=list/For each=] |rawInputIndex:long| of [=the range=] from 0 to
           |inputCount| − 1:
             <ol>
-              <li>If the the gamepad button at index |rawInputIndex|
-              [=represents a Standard Gamepad button=]:
+              <li>Determine the type of the button at index |rawInputIndex|
+              [=represents a Standard Gamepad button=] or Extended button
+              types as described in the {{ExtendedMapping}} using the provided
+              buttonTypes array. If the button does not have a canonical type,
+              use GamePadButtonType.non_canonical.
```

Comment:
This should be specified as: 

```suggestion
              <li>Let |button| be the button at index |rawInputIndex|.</li>
              <li>Let |type| be the result of determining if the |button|'s type represents a [=Standard Gamepad button|represents a Standard Gamepad button=] or an [=Extended Gamepad button|represents a Extended Gamepad button=]. If it's neither, use {{GamePadButtonType/"non_canonical"}}. 
              </li>
```

And `<dfn>represents a Standard Gamepad button</dfn>` should be defined underneath where "represents a Standard Gamepad button" is defined/